### PR TITLE
[FIX] project.task for code optimization

### PR DESCRIPTION
**Pull request by jananjoy**

### CODE REVIEW

File: project_event/models/project_task.py [hunk 1]
```diff
@@ -592,45 +593,41 @@ def reserve_equipment_inside(self, event_id):
...
-                'state': 'draft'
-            }
-        )
+            reserve_event = self.info_calendar_event()
+            reserve_event.write(
+                {'state': 'draft'}
+            )
 
     @api.multi
     def open_resources_reservation(self):
```
Comment (eilst) [inline project_event/models/project_task.py]: Looks good, you can improve it more by merging those three methods into one. "def change_resource_reservation(self, status_change)"

Comment (jananjoy) [reply project_event/models/project_task.py]: @eilst  Change according to your comment. Thx you

File: project_event/models/project_task.py [hunk 2]
```diff
@@ -652,7 +632,7 @@ def action_cancel(self):
         if self.activity_task_type == 'task' and \
                 self.task_state in ['requested', 'read', 'postponed', 'accepted']:
             self.send_message('canceled')
-            self.cancel_resources_reservation()
+            self.write_resources_reservation_state('canceled')
```
Comment (eilst) [inline project_event/models/project_task.py]: it's 'cancelled', not 'canceled'

Comment (jananjoy) [reply project_event/models/project_task.py]: @eilst  done

**Review by rimbendhaou — APPROVED** [review]

**Pull request closed without merging**